app/uk/gov/hmrc/hmrcemailrenderer/templates/offpayroll/offpayrollPrivateBetaInvitation.scala.html

```diff
@@ -0,0 +1,16 @@
+@(params: Map[String, Any])
```

Comment:
Please can you check the style guide attached, please can you add a salutation and check the format.

https://github.com/hmrc/hmrc-email-renderer/blob/master/CONTRIBUTING.md#content-recommendations